pom.xml

```diff
@@ -49,19 +49,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.confluent</groupId>
+            <groupId>org.apache.kafka</groupId>
             <artifactId>copycat-api</artifactId>
-            <version>${copycat.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>io.confluent</groupId>
-            <artifactId>copycat-runtime</artifactId>
-            <version>${copycat.version}</version>
+            <version>${kafka.version}</version>
         </dependency>
         <dependency>
             <groupId>io.confluent</groupId>
-            <artifactId>copycat-data</artifactId>
-            <version>${copycat.version}</version>
+            <artifactId>copycat-avro-converter</artifactId>
```

Comment:
One drawback to including this whole dependency is that it pulls in large dependencies -- obviously avro, copycat-api, etc. don't matter, but this will actually pull in all of Kafka including the Scala libs because `copycat-avro-converter` depends on `kafka-avro-serializer` which includes `Decoder` implementations to support the old consumer.

Should we refactor the code in schema registry at all to make `AvroData` reusable without the extra dependencies? Not sure if this is really a problem, but given the potential for conflicts and since jars for all connectors need to be loaded into the same worker processes, I'd like to try to keep dependencies to a minimum when possible.